platform-ajax/src/main/java/com/softicar/platform/ajax/testing/selenium/engine/level/low/AjaxSeleniumLowLevelTestEngineEventSimulator.java

```diff
@@ -47,6 +47,23 @@ public void simulateInput(IDomNode node) {
 		javascriptExecutor.execute(javascript, engine.getOutput().getAttributeValue(node, "id"));
 	}
 
+	@Override
+	public void simulateWheel(IDomNode node, int deltaY) {
```

Comment:
why do we need to simulate this?

Reply:
It does not. While wheel input was generally implemented with Selenium 4.2, the API thus far only allows for scrolling of the viewport / document.

Reply:
indeed, suprising